gulpfile.js

```diff
@@ -75,6 +75,66 @@ const removePrefix = (gulpRenameOptions) => {
   return gulpRenameOptions;
 };
 
+const tokenify = require('./formats/tokens');
+
+theo.registerTransform('theme', ['color/hex']);
+theo.registerFormat('light.yml', tokenify('light'));
```

Comment:
There is a bit of convention magic happening here. Each yml file in `/tokens/themes` will get processed for both light and dark. Based on the yml file name, the formatter will try and load a config file with the same filename and if it doesn’t exist will default to the `base` config. The idea is to make it really easy to extend the base theme and config to make new themes.

Reply:
This is really good, I like that! This might be what's needed to get Theo to understand the format correctly, and allow imports (which didn't work with the "2-step process" we previously discussed).